api_tests/src/shared.ts

```diff
@@ -673,6 +673,14 @@ export async function saveUserSettingsFederated(
   return await saveUserSettings(api, form);
 }
 
+// Catches a bug where when only the person or local_user changed
+export async function saveUserSettingsLimited(api: LemmyHttp) {
+  let form: SaveUserSettings = {
+    theme: "test",
+  };
+  return await saveUserSettings(api, form);
+}
```

Comment:
Doesnt make sense to have this in shared when its only used in a single test. So move it directly into the test.